robottelo/host_helpers/satellite_mixins.py

```diff
@@ -374,7 +375,9 @@ def provisioning_cleanup(self, hostname, interface='API'):
                 host[0].delete()
             assert not self.api.Host().search(query={'search': f'name={hostname}'})
         # Workaround SAT-28381
-        if not settings.server.is_ipv6:
+        if (
+            self.network_type != NetworkType.IPV6
+        ):  # TODO(sganar): What should we do in case of dualstack?
```

Comment:
```suggestion
        if (
            self.network_type == NetworkType.IPV4
        ):  # TODO(sganar): What should we do in case of dualstack?
```

Reply:
removing the TODO comment.